scene/gui/tree.cpp

```diff
@@ -3506,9 +3506,12 @@ int Tree::get_column_width(int p_column) const {
 		return columns[p_column].min_width;
 	}
 
+	int expand_area = get_size().width;
+
 	Ref<StyleBox> bg = cache.bg;
 
-	int expand_area = get_size().width - (bg->get_margin(SIDE_LEFT) + bg->get_margin(SIDE_RIGHT));
+	if (bg.is_valid())
+		expand_area -= (bg->get_margin(SIDE_LEFT) + bg->get_margin(SIDE_RIGHT));
```

Comment:
```suggestion
	if (bg.is_valid()) {
		expand_area -= (bg->get_margin(SIDE_LEFT) + bg->get_margin(SIDE_RIGHT));
	}
```

Reply:
Thanks! The 3.x branch didn't have braces for any if-statements so I just followed that